repository/Seaside-Core.package/WASessionCookieProtectionFilter.class/class/browserSessionIDCookieKey.st

```diff
@@ -0,0 +1,4 @@
+private - constants
+browserSessionIDCookieKey
+
+	^ 'seaside_browser_session'
```

Comment:
Traditionally uses uses short and cryptic key names like `_b` but this is fine as well.

Reply:
In the case of cookie names, we might want to be a bit less cryptic ;)